resources/networking_network_policies.go

```diff
@@ -0,0 +1,663 @@
+								Type:        schema.TypeStringArray,
+								Resolver:    schema.PathResolver("IPBlock.Except"),
+							},
+						},
+						Relations: []*schema.Table{
+							{
+								Name:        "k8s_networking_network_policy_ingress_from_pod_selector_match_expressions",
+								Description: "A label selector requirement is a selector that contains values, a key, and an operator that relates the key and values.",
+								Resolver:    fetchNetworkingNetworkPolicyIngressFromPodSelectorMatchExpressions,
+								Columns: []schema.Column{
+									{
+										Name:        "network_policy_ingress_from_cq_id",
+										Description: "Unique CloudQuery ID of k8s_networking_network_policy_ingress_from table (FK)",
+										Type:        schema.TypeUUID,
+										Resolver:    schema.ParentIdResolver,
+									},
+									{
+										Name:        "key",
+										Description: "key is the label key that the selector applies to. +patchMergeKey=key +patchStrategy=merge",
+										Type:        schema.TypeString,
+									},
+									{
+										Name:        "operator",
+										Description: "operator represents a key's relationship to a set of values. Valid operators are In, NotIn, Exists and DoesNotExist.",
+										Type:        schema.TypeString,
+									},
+									{
+										Name:        "values",
+										Description: "values is an array of string values",
+										Type:        schema.TypeStringArray,
+									},
+								},
+							},
+							{
+								Name:        "k8s_networking_network_policy_ingress_from_namespace_selector_match_expressions",
+								Description: "A label selector requirement is a selector that contains values, a key, and an operator that relates the key and values.",
+								Resolver:    fetchNetworkingNetworkPolicyIngressFromNamespaceSelectorMatchExpressions,
+								Columns: []schema.Column{
+									{
+										Name:        "network_policy_ingress_from_cq_id",
+										Description: "Unique CloudQuery ID of k8s_networking_network_policy_ingress_from table (FK)",
+										Type:        schema.TypeUUID,
+										Resolver:    schema.ParentIdResolver,
+									},
+									{
+										Name:        "key",
+										Description: "key is the label key that the selector applies to. +patchMergeKey=key +patchStrategy=merge",
+										Type:        schema.TypeString,
+									},
+									{
+										Name:        "operator",
+										Description: "operator represents a key's relationship to a set of values. Valid operators are In, NotIn, Exists and DoesNotExist.",
+										Type:        schema.TypeString,
+									},
+									{
+										Name:        "values",
+										Description: "values is an array of string values",
+										Type:        schema.TypeStringArray,
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			{
+				Name:        "k8s_networking_network_policy_egress",
```

Comment:
same ingress above I think an extra table was created

Reply:
there is ingress and egress - they are different